src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -56,90 +57,107 @@ const ActiveState = ({
     getApproverWithQuery,
     getAvatarUrl,
     getUserProfileUrl,
-}: Props): React.Node => (
-    <ul className="bcs-activity-feed-active-state">
-        {items.map((item: any) => {
-            const { type, id, versions, permissions } = item;
-            const isFocused = activeFeedEntryId === id && activeFeedEntryType === type;
-            const refValue = isFocused ? activeFeedItemRef : undefined;
+}: Props): React.Node => {
+    const activeEntriesInFeed = items.filter(item => {
```

Comment:
This implies there can be more than one entry active at a time, which I don't think is possible. Should this read:

```
const activeEntry = items.find(({ id, type }) => (
    id === activeFeedEntryId && type === activeFeedEntryType && isValidType(type)
));
```